applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -16,6 +16,14 @@
 
 #define LE_AUDIO_SDU_SIZE_OCTETS(bitrate) (bitrate / (1000000 / CONFIG_AUDIO_FRAME_DURATION_US) / 8)
 
+#if (CONFIG_SCAN_MODE_ACTIVE)
+#define BT_LE_SCAN_TYPE    BT_LE_SCAN_TYPE_ACTIVE
+#define BT_LE_SCAN_PARAMS  BT_LE_SCAN_ACTIVE
+#else
```

Comment:
Some nitpick, but these changes make the code more resilient from outside/upstream changes.
```suggestion
#elif (CONFIG_SCAN_MODE_PASSIVE)
```
And then an #error .

Reply:
Can add that, too.